### ISSUE TITLE
Feature Request: Option to Import Already Downloaded Epic Store Games into Junk Store

### ISSUE THREAD

**Issue by shaka456** [ebenbruyns/junkstore#63]
**Is your feature request related to a problem? Please describe.** 

I have games already downloaded from the Epic Store on Windows, and I'm unsure if there is a way to import them into Junk Store.

**Describe the solution you'd like**

It would be great to have a feature that allows users to import existing Epic Store games into Junk Store, performing the necessary checks to add and run the game correctly.

**Describe alternatives you've considered**

I've considered manually copying the game files, but Junk Store doesn't recognize the games when you do this. It would be helpful to have a button that allows importing the game, performs the necessary verifications, and adds it to the library.

**Additional context**

This feature would help users transition more smoothly from other platforms and save time by avoiding re-downloads.

**Comment by Thaunier** [comment]
I also agree on this one; it’d be great to migrate installed games :) I had a few games already installed before finding Junk Store, so it’d just make the find sweeter :)

**Comment by mrsjunkrunner** [comment]
This feature is not feasible to implement in the open source version. However, it is currently in QA in Junk Store Pro and due to be released at some point in the near future.

We don't have the manpower or resources to build this into the opensource version of Junk Store.  If the community wants to help work on this feature for the the open source version we can reopen it.
